samples/FakeItEasy.Examples.ruleset

```diff
@@ -15,4 +14,4 @@
     <Rule Id="CA2201" Action="None" />
     <Rule Id="CA2210" Action="None" />
   </Rules>
-</RuleSet>
\ No newline at end of file
+</RuleSet>

```

Comment:
> May have been emacs.

What?! You're not using vim?
(I didn't think it was possible for Windows users to have this debate, but apparently it is  :laughing:)